src/cart/crt_internal_types.h

```diff
@@ -220,6 +220,8 @@ struct crt_event_cb_priv {
 	ENV(SWIM_PING_TIMEOUT)                                                                     \
 	ENV(SWIM_PROTOCOL_PERIOD_LEN)                                                              \
 	ENV(SWIM_SUSPECT_TIMEOUT)                                                                  \
+	ENV(FI_TCP_PORT_LOW_RANGE)                                                                 \
+	ENV(FI_TCP_PORT_HIGH_RANGE)                                                                \
```

Comment:
I don't understand why putting this in other order would help (saw your last patch).

Reply:
No idea, but i had to do it this way for the build to pass. @frostedcmos any idea?

Reply:
Hmm not sure. I submitted a PR in the order that caused you issues in run #3 and it passed for me:
 https://github.com/daos-stack/daos/pull/15228

Could there be some whitespace somewhere, resulting in bad macro expansion in previous tries? I can't easily tell in github views.

Reply:
maybe